packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -352,3 +323,52 @@ class NetworkController extends DisposableController
     }
   }
 }
+
+class CurrentNetworkRequests {
+  CurrentNetworkRequests({required this.onDataUpdate});
+
+  Map<String, NetworkRequest> requestsMapping = <String, NetworkRequest>{};
```

Comment:
nit: rename to `requestsById` so it is clear from the name what the map is between